src/v0/destinations/active_campaign/transform.js

```diff
@@ -89,13 +92,14 @@ const customTagProcessor = async (message, category, destination, contactId) =>
   // Step - 1
   // Fetch already created tags from dest, so that we avoid duplicate tag creation request
   // Ref - https://developers.activecampaign.com/reference/retrieve-all-tags
-  endpoint = `${destination.Config.apiUrl}${`${category.tagEndPoint}?limit=100`}`;
+  endpoint = `${destination.Config.apiUrl}${`${tagEndPoint}?limit=100`}`;
```

Comment:
do we want to put limit = 100 in endpoint?

Reply:
It was already there, this was added long time back as a pagination support

Reply:
but do we want pagination params part of the metrics?

Reply:
no but I passed tagEndPoint in the metric right, so it wouldn't have pagination params right?

Reply:
I got confused.